oauth2client/contrib/django_util/models.py

```diff
@@ -16,6 +16,12 @@
 
 import base64
 import pickle
+import sys
+
+try:
+    import jsonpickle
+except ImportError:
+    pass
```

Comment:
jsonpickle is not installed with the package, but I'd prefer to do that if it's okay.